randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -398,9 +399,11 @@ Extra - room_id: [13, 46]
           All of the following:
```

Comment:
Central shaft:
beside drop -> door to reo room needs to account for needing to mid air morph/spring ball to break the bomb blocks with bombs only